### ISSUE TITLE
[JavaScript] Inconsistent scope for class properties

### ISSUE THREAD

**Issue by predragnikolic** [sublimehq/Packages#1700]
![2018-08-30-080749_683x384_scrot](https://user-images.githubusercontent.com/22029477/44832694-d3b18d80-ac2b-11e8-979d-ed4682b0afb4.png)

Here are the scopes for the given properies:

```javascript
class Person {
  constructor(name, age, type, hair, height, weight, eyes) {
    this.name = name;
    //   ^^^^  source.js meta.class.js meta.block.js meta.block.js support.constant.dom.js
    this.age = age;
    //   ^^^  source.js meta.class.js meta.block.js meta.block.js meta.property.object.js
    this.type = type;
    //   ^^^^  ... support.constant.dom.js
    this.hair = hair;
    //   ^^^^  ... meta.property.object.js
    this.height = height;
    //   ^^^^^^  ... support.constant.dom.js
    this.weight = weight;
    //   ^^^^^^  ... meta.property.object.js
    this.eyes = eyes;
    //   ^^^^  ... meta.property.object.js
  }
}
```
Apparently github follows the same rules for determining scopes :D

**Comment by michaelblyons** [comment]
These are my guesses, from MDN:
- Web/API/Element/[name][]
- Web/SVG/Attribute/[type][]
- Web/SVG/Attribute/[height][]

Why don't you have a [width][] on your `Person` instead of a weight? 😉

[name]: https://developer.mozilla.org/en-US/docs/Web/API/Element/name
[type]: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/type
[height]: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/height
[width]: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/width

**Comment by Thom1729** [comment]
This is an old feature of the JavaScript syntax that specially highlights some property names that are used in the DOM API. [This PR](https://github.com/sublimehq/Packages/pull/1641) will remove those.

**Comment by predragnikolic** [comment]
@Thom1729 Thanks for replying, awsome :D
@michaelblyons hahaha, I think I could go with `width`.
Here is my new "implementation". Hair was a problem. Luckily functional programming came to the rescue :D
![2018-08-30-160025_835x529_scrot](https://user-images.githubusercontent.com/22029477/44856618-29a72500-ac6e-11e8-9564-e3424af8413f.png)

**Comment by Thom1729** [comment]
Fixed by #1641.
